apps/nextjs/src/app/layout.tsx

```diff
@@ -94,7 +94,7 @@ export default async function RootLayout({
             grayColor="olive"
             scaling="110%"
             color="#22222"
```

Comment:
Should we delete this line, or explain why it's still needed if overridden below?